fluss-server/src/test/java/com/alibaba/fluss/server/kv/snapshot/TestCompletedSnapshotHandleStore.java

```diff
@@ -42,7 +39,6 @@ public TestCompletedSnapshotHandleStore(
                     getAllSupplier,
```

Comment:
`SupplierWithException<List<Tuple2<CompletedSnapshotHandle, String>>, Exception>
                    getAllSupplier` can also be removed..

Reply:
ok, done